### ISSUE TITLE
SC2216 shouldn't apply to nice

### ISSUE THREAD

**Issue by brianjmurrell** [koalaman/shellcheck#1169]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2216
- My shellcheck version (`shellcheck --version` or "online"): 0.4.7
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash

echo Yes | nice bash -c "read a; echo \$a"
```

#### Here's what shellcheck currently says:
```
Line 3:
echo Yes | nice bash -c "read a; echo \$a"
           ^-- SC2216: Piping to 'nice', a command that doesn't read stdin. Wrong command or missing xargs?
```

#### Here's what I wanted or expected to see:
I don't think SC2216 applies to `nice` since the above script produces:

```sh
# echo Yes | nice sh -c "read a; echo \$a"
Yes
```

**Comment by koalaman** [comment]
What a glaring oversight! This has now been fixed, thanks.
